src/client/sky.cpp

```diff
@@ -892,19 +831,57 @@ void Sky::setStarCount(u16 star_count, bool force_update)
 	// Allow force updating star count at game init.
 	if (m_star_params.count != star_count || force_update) {
 		m_star_params.count = star_count;
-		m_stars.clear();
-		// Rebuild the stars surrounding the camera
-		for (u16 i = 0; i < star_count; i++) {
-			v3f star = v3f(
-				myrand_range(-10000, 10000),
-				myrand_range(-10000, 10000),
-				myrand_range(-10000, 10000)
-			);
-
-			star.normalize();
-			m_stars.emplace_back(star);
-		}
+		m_seed = (u64)myrand() << 32 | myrand();
+		updateStars();
+	}
+}
+
+void Sky::updateStars() {
```

Comment:
```suggestion
void Sky::updateStars()
{
```
(in a few other places too)

Reply:
found only one other place